app/controllers/user.go

```diff
@@ -127,6 +127,12 @@ func (c Users) ListLocations() revel.Result {
 	return c.Data(locations)
 }
 
+func (c Users) Test() revel.Result {
```

Comment:
We shouldn't add tests in the controller. And it's not used in the apptest, right?